docs/source/index.rst

```diff
@@ -1,66 +1,16 @@
-Welcome to Aryn!
+Welcome to Sycamore!
 ================
-Aryn is an LLM-powered data preparation, processing, and analytics system for complex, unstructured documents like PDFs, HTML, presentations, and more. With Aryn, you can prepare data for GenAI and RAG applications, power high-quality document processing workflows, and run analytics on large document collections with natural language. It includes two components: The Aryn Partitioning Service and Sycamore.
-
-The Aryn Partitioning Service (APS) is a serverless, GPU-powered API for segmenting and labeling PDF documents, doing OCR, and extracting tables and images. It returns the output in JSON. APS runs the Aryn Partitioner and its `state-of-the-art, open source deep learning AI model <https://huggingface.co/Aryn/deformable-detr-DocLayNet>`_ trained on 80k+ enterprise documents. You can use it to partition documents and extract information directly in your code, or use it with Sycamore for additional processing. `Sign-up here for free <https://www.aryn.ai/get-started>`_ to get an API Key and use the `Aryn Playground <https://play.aryn.cloud/partitioning>`_ to visually see how it segments and processes your own documents. Or, watch the Aryn Partitioning Service in action in `this video <https://www.aryn.ai/?name=ArynPartitioningService_Intro>`_.
+Aryn is an LLM-powered data preparation, processing, and analytics system for complex, unstructured documents like PDFs, HTML, presentations, and more. With Aryn, you can prepare data for GenAI and RAG applications, power high-quality document processing workflows, and run analytics on large document collections with natural language. It includes three components: Aryn DocParse, Aryn DocPrep, and Sycamore. For Aryn DocParse and Aryn DocPrep, please visit the `Aryn DocParse documentation <https://docs.aryn.ai>`_.
```

Comment:
I would remove this paragraph. I would then add: "For Aryn DocParse and Aryn DocPrep, which use Sycamore, please visit the `Aryn documentation <https://docs.aryn.ai>`_." I would put this after the paragraph with Qdrant and more.